pkg/cmd/extension/command.go

```diff
@@ -60,6 +60,7 @@ func NewCmdExtension(f *cmdutil.Factory) *cobra.Command {
 
 					t.AddField(fmt.Sprintf("gh %s", c.Name()), nil, nil)
 					t.AddField(repo, nil, nil)
+					t.AddField(c.CurrentVersion(), nil, nil)
```

Comment:
Lets do the truncation of the current version SHA into a short SHA for non-binary extensions here and keep it to 8 characters for now.

Reply:
I didn't know that the length of 'rev-parse --short' is variable. https://git-scm.com/docs/git-config#Documentation/git-config.txt-coreabbrev
I thought trimming it to a fixed 8-character length would not be a problem, so I simply trimmed the SHA.
https://github.com/cli/cli/pull/5219/commits/f48468db300c7b3e27205f3c43b2de891568601e#diff-a3988c516ff46e7198c54a896c3102f8862f7173adfb40fce8c516cb0eee8bf3R65